dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKServer/resources/logging/logging.properties

```diff
@@ -0,0 +1,37 @@
+#Handlers we plan to use
+handlers=java.util.logging.FileHandler,java.util.logging.ConsoleHandler
+
+#Global logger - By default only log warnings
+.level=WARNING
+
+#Concurrency logger - By default log everything for concurrency loggers
+ee.jakarta.concurrent.level=ALL
```

Comment:
Is this the correct package name?  I thought we switched to ee.jakarta.tck.concurrent?